src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -99,8 +103,8 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
 
 
     //TODO: Thread safe version
-    let fixes = System.Collections.Generic.Dictionary<DocumentUri, (LanguageServerProtocol.Types.Range * TextEdit) list>()
-    let analyzerFixes = System.Collections.Generic.Dictionary<(DocumentUri * string), (LanguageServerProtocol.Types.Range * TextEdit) list>()
+    let lintFixes = System.Collections.Generic.Dictionary<DocumentUri, (LanguageServerProtocol.Types.Range * TextEdit) list>()
+    let analyzerFixes = System.Collections.Generic.Dictionary<DocumentUri, System.Collections.Generic.Dictionary<string, (LanguageServerProtocol.Types.Range * TextEdit) list>>()
```

Comment:
made this a doubly-nested dictionary to make the common case of 'give me all analyzer results for file 'X'' easier to accomplish.